finals-2024-01-16/code/5311-kristijan-lazarov/src/app/models/band.ts

```diff
@@ -0,0 +1,23 @@
+export interface Bands {
```

Comment:
The model names should be singular (Band not Bands)
+5 points for the models, the definitions are ok.